orc8r/tools/fab/pkg.py

```diff
@@ -131,11 +131,11 @@ def upload_pkgs_to_aws():
 
 
 def get_magma_version():
-    return run('ls ~/magma-packages'
-               ' | grep "^magma_[0-9].*"'
-               ' | xargs -I "%" dpkg -I ~/magma-packages/%'
-               ' | grep "Version"'
-               ' | awk \'{print $2}\'')
+    return run('directory=$(mktemp -d) &&'
+               'dpkg-deb --extract ~/magma-packages/magma_[0-9]*.deb $directory &&'
```

Comment:
Maybe we can ignore this one. https://wemake-python-stylegui.de/en/0.16.0/pages/usage/configuration.html#configuration

Reply:
Sure. Inserted an inline-comment.